.github/workflows/inspector_checks.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: RxPaired Inspector Checks
+name: RxPaired Checks
```

Comment:
This file is named "inspector_checks" but it does check on all the repo, and the file named "checks" does check only for folder `/inspector`